docs/developer/tutorials/snmp/sim-format.md

```diff
@@ -15,7 +15,7 @@ Lines must be formatted as follows:
 <OID>|<type>|<value>
 ```
 
-For the list of supported types, see the [`snmpsim` simulation data file format](http://snmplabs.com/snmpsim/managing-simulation-data.html#file-format) documentation.
+For the list of supported types, see the [`snmpsim` simulation data file format](https://github.com/etingof/snmpsim/blob/69751eb81381ce1fe89bc1cd1154f9daaa5b697b/docs/source/documentation/managing-simulation-data.rst#file-format) documentation.
```

Comment:
Why not https://github.com/etingof/snmpsim/blob/master/docs/source/documentation/managing-simulation-data.rst
 ?

Reply:
To have link focused as closed as possible to the type format definition. I'm not against redirecting to the page's root